### DISCUSSION TITLE
Lokinet as a transport of Locutus

### DISCUSSION THREAD

**Discussion by ple1n** [oxen-io/lokinet#2186] [General]
It's feasible, but as to whether we should do it, I think, replication is unavoidable and necessary for decentralization, anonymity is better offered by a specialized network of forwarders (persistent anonymous storage is hated), an anonymity network provides E2E connection with both sides anonymized (which is maximally anonymous).

It overlays Locutus on Lokinet. Optimization may be done by reducing 2 DHT lookups into 1 (both overlay networks have their own DHT and address resolution) not sure about the details.

Locutus doesn't carry a lot of data. It's kind of a cache. Trade space/bandwidth/compute off for anonymity/decentralization/security. Although solo servers running on lokinet offer enough decentralization (no single point to be taken down), Locutus can be a plus.

**Comment by majestrate** [comment]
On Thursday, 1 June 2023 04:57:20 EDT plein wrote:
> It's feasible, but as to whether we should do it, I think, replication is
> unavoidable and necessary for decentralization, anonymity is better offered
> by a specialized network of forwarders (persistent anonymous storage is
> hated), an anonymity network provides E2E connection with both sides
> anonymized (which is maximally anonymous).
> 
> It overlays Locutus on Lokinet. Optimization may be done by reducing 2 DHT
> lookups into 1 (both overlay networks have their own DHT and address
> resolution) not sure about the details.

do you have a link to the project page for locutus? 

> 
> Locutus doesn't carry a lot of data. It's kind of a cache. Trade
> space/bandwidth/compute off for anonymity/decentralization/security.
> Although solo servers running on lokinet offer enough decentralization (no
> single point to be taken down), Locutus can be a plus.


-- 
~jeff
